src/models/checkout-object/checkout-object-data-mapper-impl.ts

```diff
@@ -79,12 +80,14 @@ export class CheckoutObjectDataMapperImpl extends GenericDataMapper
       .toPromise();
   }
 
-  public async getAll(opts?: IUowOpts): Promise<IDbResult<CheckoutObject[]>> {
+  public async getAll(opts?: IUowOpts): Promise<IDbResult<ModifiedCheckoutObject[]>> {
```

Comment:
Do we have to name it ModifiedCheckoutObject? Maybe we can think of something better.